projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/payment-methods.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Payment Methods Page accessibility', { testIsolation: false }, () => {
+  before(() => {
+    cy.a11yContinuumSetup();
+    cy.login('test-user-with-orders@sap.cx.com', 'pw4all');
+    cy.visit('/my-account/payment-details');
```

Comment:
When I logged in with this user, there were no existing payment methods available. Did you create this user or find it from somewhere? Any new user would need sample data created so I'd check out using the standard user from sampledata to see if it has some payment details already there.